### ISSUE TITLE
where does not work

### ISSUE THREAD

**Issue by dzuluaga** [cesardeazevedo/sails-hook-sequelize-blueprints#12]
Great hook. Pretty much everything worked except for **find where** clause. For instance, I tried to run the following API request and it returned a strage error:

``` http
GET /user?where={"name":{"contains":"Eri"}} HTTP/1.1
Host: localhost:1337
```

**Error:**

```
{
  "error@context": {}
}
```

Am I doing anything wrong? Thanks!

**Comment by dzuluaga** [comment]
I've just found the solution. Instead of using contain, which is the notation from Waterline. You need to stick to sequelize notation. Therefore, the search above should be translated into:

```
curl -G -X GET 'http://localhost:1337/user' --data-urlencode 'where={"name":{"$like":"%Eri%"}}' -v
```

**Comment by cesardeazevedo** [comment]
Interesting, it's related to the waterline / sequelize notation or something with this hook? i haven't tested with the `where` criteria object, only with `http://localhost:1337/user?name=value&age=21`, 

thanks for the feedback.

**Comment by dzuluaga** [comment]
Correct, my confusion was that I was reading Sails.js Blueprint and I was using the exact Waterline notation to run "like" queries, and the response from the server wasn't helping to troubleshoot the issue. After I looked into [the code in this line](https://github.com/cesardeazevedo/sails-hook-sequelize-blueprints/blob/master/actions/find.js#L39), I realized that it used sequelize model, which uses a sligthly different convention than waterline. I wonder that more users may have run into this issue and this could save them some time of debugging.

**Comment by lahvey** [comment]
Thanks @dzuluaga, it works by using sequelize combinations. By the way, the query string should be encoded like `--data-urlencode`, such as `http://localhost:1337/user?where={"$or":[{"name":{"$like":"hello%25"}},{"id":2}]}`, where `%25` denotes to `%`, otherwise the where condition should not work.

**Comment by loulin** [comment]
Thanks @dzuluaga, it works by using sequelize combinations. By the way, the query string should be encoded like `--data-urlencode`, such as `http://localhost:1337/user?where={"$or":[{"name":{"$like":"hello%25"}},{"id":2}]}`, where `%25` denotes to `%`, otherwise the where condition should not work.
